test/backend.js

```diff
@@ -152,7 +152,8 @@ describe('Full walkthrough', function () {
     })
 
     it('Should open sockets for Alice and Bob', async function () {
-      for (let user of Object.values(users)) {
+      const userList = Object.keys(users).map((userKey) => users[userKey])
+      for (let user of userList) {
```

Comment:
Changed it to the Object.keys() version only because it fixes the flow error even though using Object.values() is obviously simpler and cleaner.
please refer to :

facebook/flow#2221
https://github.com/facebook/flow/blob/master/lib/core.js#L65